org.hl7.fhir.utilities/src/main/java/org/hl7/fhir/utilities/npm/FilesystemPackageCacheManager.java

```diff
@@ -521,26 +521,20 @@ private InputStream fetchFromUrlSpecific(String source, boolean optional) throws
 
   private InputStreamWithSrc loadFromCIBuild(String id) throws IOException {
     checkBuildLoaded();
-    if (ciList.containsKey(id)) {
-      InputStream stream = fetchFromUrlSpecific(Utilities.pathURL(ciList.get(id), "package.tgz"), false);
-      return new InputStreamWithSrc(stream, Utilities.pathURL(ciList.get(id), "package.tgz"), "current");
+    if (packageRepos.containsKey(id)) {
+      InputStream stream = fetchFromUrlSpecific(Utilities.pathURL((String)packageRepos.get(id), "package.tgz"), false);
+      return new InputStreamWithSrc(stream, Utilities.pathURL((String)packageRepos.get(id), "package.tgz"), "current");
```

Comment:
Do you actually need to cast this to a String? The HashMap is typed already?

Reply:
Fixed